fontc/src/main.rs

```diff
@@ -390,10 +402,28 @@ impl ChangeDetector {
     fn new(config: Config, ir_paths: IrPaths, prev_inputs: Input) -> Result<ChangeDetector, Error> {
         // What sources are we dealing with?
         let mut ir_source = ir_source(&config.args.source)?;
-        let current_inputs = ir_source.inputs().map_err(Error::FontIrError)?;
+        let mut current_inputs = ir_source.inputs().map_err(Error::FontIrError)?;
         let be_paths = BePaths::new(ir_paths.build_dir());
 
+        let glyph_name_filter = config
+            .args
+            .glyph_name_filter
+            .map(|raw_filter| Regex::new(&raw_filter))
+            .transpose()
+            .map_err(Error::BadRegex)?;
+
+        if let Some(regex) = &glyph_name_filter {
+            current_inputs.glyphs.retain(|glyph_name, _| {
```

Comment:
Per im, feature compilating requires the full glyph list. Assuming we want to eventually support filtering in combination with features (compile time subsetting?) it might be worth thinking about how we retain the original glyph order and map between that and the filtered set.

Reply:
We can store it on Context if it turns out we need it.